slu-scripts/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import yargs from 'yargs';
+import { encryptData, decryptData, sendAuthProof } from './auth-proof/auth-proof';
+
+const argv = yargs
+	.command('encrypt', 'Encrypt for encrypting')
```

Comment:
Maybe also add which params are possible and what they mean like --key-file (path to the key) --input (path to input data) --dest (path to encrypted output)

same for send-proof